src/generate-method-mapping.js

```
@@ -1,23 +1,50 @@
 const jsf = require('json-schema-faker');
+const Ajv = require('ajv');
+const _ = require('lodash');
 
-const generateResponse = async (schema, methodName) => {
-  const method = schema.methods[methodName];
-
-  if (method === undefined) { return 'method not found: ' + methodName; }
+const generateResponse = async (method) => {
+  if (method === undefined) { return 'method not found: ' + method.name; }
```

should this throw or something instead of return a string?

---

#18